query/app/src/main/resources/features.xml

```diff
@@ -48,12 +50,24 @@
     <feature name="admin-query-wcpm" install="manual" version="${project.version}"
              description="DDF :: Admin Console :: Query :: Security :: Web Context Policy Manager">
         <feature>admin-query-core</feature>
+        <feature>admin-query-utils</feature>
+        <feature>admin-query-wcpm-dependencies</feature>
         <bundle>mvn:org.codice.ddf.admin.query/admin-query-wcpm/${project.version}</bundle>
     </feature>
 
+    <feature name="admin-query-wcpm-dependencies" install="manual" version="${project.version}"
+             description="DDF :: Admin Console :: Query :: Security :: Web Context Policy Manager Dependencies">        <bundle>mvn:commons-collections/commons-collections/3.2.2</bundle>
+        <bundle>mvn:commons-collections/commons-collections/3.2.2</bundle>
```

Comment:
Yes, this is the app we're using in production. DDF won't have the thirdparty feature we're using in the comp tests.